src/features/withdraw/components/WithdrawForm/selectors.ts

```diff
@@ -0,0 +1,37 @@
+import type { SnapshotFrom } from "xstate"
+import { isAggregatedQuoteEmpty } from "../../../../services/quoteService"
+import type { withdrawUIMachine } from "../../../machines/withdrawUIMachine"
+
+/**
+ * @return null | boolean - null if not enough info to determine
+ */
+export function isLiquidityUnavailable(
```

Comment:
Yes, we check it like this. `false` means liquidity is enough, on other hand `null` means we don't know and can't make informed decision, it might not be important for this particular case, but in other case it might be important to distinguish these values, code should not make an assumption or lie to developers as it may cause unexpected issues.